src/browser/web_request_handler.ts

```diff
@@ -84,10 +84,10 @@ function beforeSendHeadersHandler(details: RequestDetails, callback: (response:
         if (wc) {
             electronApp.vlog(1, `${moduleName}:beforeSendHeadersHandler got webcontents ${wc.id}`);
             const bw = wc.getOwnerBrowserWindow();
-            if (bw) {
+            if (bw && bw.id) {
```

Comment:
Is there ever a chance that the BrowserWindow id is going to be `0`? Maybe safer to check `typeof`